apps/docs/scripts/test-template.ts

```diff
@@ -11,7 +11,7 @@ afterAll(() => {
 });
 
 // %TEST_ENVIRONMENT%
-test('%NAME%', async () => {
+test('%NAME%', { timeout: 10_000 }, async () => {
```

Comment:
The [`wrong-resubmission` snippet](https://github.com/FuelLabs/fuels-ts/blob/f260c71d491d75eff6310fe1c9b23bb3ddaddefa/apps/docs/src/guide/cookbook/snippets/resubmitting-failed-transactions/wrong-resubmission.ts#L32) test was failing with a timeout because transactions started staying in the TxPool for some time in the new version, which wasn't happening previously. This timeout covers such cases in our snippets.